redis-worker/src/config.ts

```diff
@@ -0,0 +1,17 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const config = {
+  redis: {
+    host: process.env.REDIS_HOST,
+    port: process.env.REDIS_PORT,
+    key: process.env.REDIS_KEY,
+  },
+  postgrest: {
+    url: `http://${process.env.POSTGREST_ENDPOINT}/medic`, // TODO: Make this dynamic to handle multiple DBs
```

Comment:
Created an issue to handle multiple DBs -> https://github.com/medic/cht-sync/issues/86